include/amici_model.h

```diff
@@ -6,6 +6,10 @@
 #include <sundials/sundials_direct.h>
 #include <sundials/sundials_sparse.h>
 
+int fdJdx(realtype t, N_Vector x, N_Vector dx, void *user_data);
+int fdJdp(realtype t, N_Vector x, N_Vector dx, void *user_data);
+int fddxdotdpdp(realtype t, N_Vector x, N_Vector dx, void *user_data);
```

Comment:
These functions need to become virtual member functions of `Model` with an empty default implementation, otherwise linking non-second-order models won't work.